student-software-developers/administration-handbook/recruitment.md

```diff
@@ -0,0 +1,4 @@
+### Background on CS departments and advertising strategies
+
+* There are 2 ways to major in Computer Science at Princeton. You can apply and matriculate to the School of Engineering and Applied Science, or you can do the major through the standard BA program. The Undergraduate Affairs Administrator for the School of Engineering, [Traci Miller](https://engineering.princeton.edu/engage/leadership-and-staff), has also posted our position to the Princeton engineering student job listserv in the past. However, an email from Colleen should reach those students as well.
+* In the School of engineering students declare their majors at the end of their freshman year. Students in the BA program declare majors at the end of their sophomore year.
```

Comment:
We have a document about recruiting ideas for front-end positions, could we add a link to that? https://docs.google.com/document/d/1RzXdnI8shNxOkWb-LgJKrAVH6p5o-jYq5lmvs-fHMVs/edit